src/Sarif.Viewer.VisualStudio/ErrorList/ErrorListService.cs

```diff
@@ -377,7 +377,10 @@ internal static async System.Threading.Tasks.Task ProcessSarifLogAsync(SarifLog
 
             if (hasResults)
             {
-                SdkUIUtilities.ShowToolWindowAsync(new Guid(ToolWindowGuids80.ErrorList), activate: false).FileAndForget(Constants.FileAndForgetFaultEventNames.ShowErrorList);
+                if (!SarifViewerPackage.IsUnitTesting) // We cannot show UI during unit-tests.
```

Comment:
>tests [](start = 88, length = 5)

Yup. In future we'd want to fire an event that some UI service could consume, but this is just fine for now.